salt/config/__init__.py

```diff
@@ -1173,7 +1173,7 @@ def _gather_buffer_space():
             salt.syspaths.BASE_FILE_ROOTS_DIR, "win", "repo-ng"
         ),
         "winrepo_cachefile": "winrepo.p",
-        "winrepo_cache_expire_max": 21600,
+        "winrepo_cache_expire_max": 604800,
```

Comment:
This pull request title was about updating Docs, not changing default values.  As the author of the caching code this should not have been changed..... from the default of 6 hours, which is less than a business day to 7 days.  It will force people people to run refresh_db, even if they have run an high-state.  FYI @twangboy

Reply:
Also need to consider some devices will not be running when one one needs to run refresh_db. It's easy for people to roll out a config file to the minion to change this setting.   The impact of this is Clients will not see new versions of software, and may install the wrong software.  FYI @dwoz

Reply:
FYI @saltstack/team-windows

Reply:
Mmm I suspect this para needs fixing. @saltstack/team-windows 
Software package definitions are automatically refreshed if stale after winrepo_cache_expire_max. Running a highstate normal forces the refresh of the package definition and generation of the metadata, unless the metadata is younger than winrepo_cache_expire**_min**.

With winrepo_cache_expire_max set to 7 days, people will get inconsistent times. Lets say you run `salt * pkg.list_pkgs`  half the minions might return in under 20 seconds (<7d) and the other half return in under 2 minutes (>7d).  An hour letter another minion could it the >7d.   The end result is the first run of  `salt * pkg.list_pkgs`  will still take 2 minutes, as its the slowest minion that holds up the results. And for the rest of the day if your unlucky you might get more minions go past the 7 day mark.    And if one is old than 7d, people will think why is this slow now?

Set to 6 hours the timing of the results is consistent and the output of the results is consistent.  Next time its run it will be quicker for all minions.   The cache was design so people running the commands had consistent results, and if they ran it multiple times within 6 hour period the results would come back faster (we are assuming they have not change the package definition files for 6 hour period).

The their the impact to `list_available` which could return different results.